examples/petsc/dmswarm.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and other CEED contributors.
```

Comment:
Note - probably should be in the `qfunctions` folder.

Reply:
Moved in refactor